services/offline/src/lib/online-status.ts

```diff
@@ -1,6 +1,15 @@
 import debounce from 'lodash/debounce'
 import { useState, useEffect, useCallback } from 'react'
 
+interface OnlineStatusOptions {
+    debounceDelay?: number
```

Comment:
To leverage types as documentation, a type alias can be used to clarify that `debounceDelay` is in milliseconds:

```typescript
type milliseconds = number
interface OnlineStatusOptions {
    debounceDelay?: milliseconds
}
```

Reply:
Nice idea :)